### ISSUE TITLE
Delay before running tests

### ISSUE THREAD

**Issue by aramk** [arunoda/laika#146]
There's about a 1 minute delay after "Ok, everything's ready. Here comes Meteor!" appears and the server starts up. There's no CPU activity during this time and it then resumes normally with "[app touch log] => Started proxy."

I noticed this issue after updating to 0.9.4 and it might be causing either a timeout or an unintended wait.

**Comment by apendua** [comment]
@aramk Good catch! Can you provide more details on how to reproduce this error?

**Comment by aramk** [comment]
I think it's actually resolved now - I haven't been having this issue with Meteor 1.0.
